src/Orleans/IDs/Interner.cs

```diff
@@ -246,10 +246,14 @@ private void InternCacheCleanupTimerCallback(object state)
                 }
             }
             long numRemoved = numEntries - internCache.Count;
-            if (numRemoved>0)
+            if (numRemoved > 0)
+            {
                 if (logger.IsVerbose) logger.Verbose(ErrorCode.Runtime_Error_100296, "Removed {0} / {1} unused {2} entries in {3}", numRemoved, numEntries, internCacheName, clock.Elapsed);
+            }
             else
+            {
                 if (logger.IsVerbose2) logger.Verbose2(ErrorCode.Runtime_Error_100296, "Removed {0} / {1} unused {2} entries in {3}", numRemoved, numEntries, internCacheName, clock.Elapsed);
```

Comment:
I think this statement was meant to be executed if condition logger.IsVerbose is false, and not if numRemoved > 0 is false. Also it will cause merge conflict with https://github.com/dotnet/orleans/pull/1444/files